airflow/operators/dagrun_operator.py

```diff
@@ -39,8 +38,8 @@ class TriggerDagRunLink(BaseOperatorLink):
     name = 'Triggered DAG'
 
     def get_link(self, operator, dttm):
-        view = conf.get('webserver', 'dag_default_view').lower()
-        return f"/{view}?dag_id={operator.trigger_dag_id}&root=&execution_date={quote(dttm.isoformat())}"
+        query = {"dag_id": operator.trigger_dag_id, "root": "", "execution_date": dttm.isoformat()}
```

Comment:
Do we need to specify `"root": ""` everytime if it is blank.

If so maybe just add it to `build_airflow_url_with_query` to add `"root": ""` if `root` key isn't present in the dict (query).